tools/objc/BUILD

```diff
@@ -16,6 +16,14 @@ sh_binary(
     data = [
         ":make_hashed_objlist.py",
         ":xcrunwrapper",
+        ":libtool_check_unique",
+    ],
+)
+
+cc_binary(
+    name = "libtool_check_unique",
```

Comment:
I think you need to set this up in the toolchain setup for this to work

Reply:
yeah i was wondering if some setup was needed. can you elaborate?

Reply:
well if this works I guess it's fine, but tools like this are normally setup here https://github.com/bazelbuild/bazel/blob/39bc97eab295bddb35b38bfc4a2ff3d2b15d034e/tools/cpp/osx_cc_configure.bzl#L52